.github/workflows/test.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Test
+
+on: push
```

Comment:
Should we limit the scope a little? Maybe we should run this on "push to an open PR" and "on push to Master (merge)"? Just a thought.

Reply:
Maybe we leave it as is and see how it goes.

Reply:
I don't see a downside to running tests on each pushed commit.  Which are you worried about?

Reply:
I was thinking that this should only run on when we're pushing to a PR.

So if we're just pushing up a commit to a working branch, then it wouldn't trigger. But I think you're right. There isn't really any downside to running it on each commit.